WordPress/Classes/ViewRelated/Post/PostCompactCell.swift

```diff
@@ -83,6 +86,12 @@ class PostCompactCell: UITableViewCell, ConfigurablePostView {
         contentView.backgroundColor = innerView.backgroundColor
     }
 
+    private func setupSeparator() {
+        WPStyleGuide.applyBorderStyle(separator)
+        contentView.heightAnchor.constraint(equalToConstant: contentView.frame.height + WPStyleGuide.separatorHeight).isActive = true
```

Comment:
I'm not sure this should be necessary, if you ensure the InnerView is pinned to the top of the content view, and the bottom of the InnerView is pinned to the top of the separator?

Reply:
This code was adding the separator height to the cell height and realign the labels (to maintain the same margins as before the hardcoded separator). But yeah, complex and not so clear.

I removed it and managed to achieve the same result with a heigh constant in the xib.